packages/account/src/Components/poi/poi-confirm-with-example-form-container/poi-confirm-with-example-form-container.tsx

```diff
@@ -134,30 +137,19 @@ const PoiConfirmWithExampleFormContainer = ({
 
     return (
         <Formik initialValues={form_initial_values} enableReinitialize onSubmit={onSubmit} validate={validateFields}>
-            {({
-                values,
-                errors,
-                touched,
-                handleChange,
-                handleBlur,
-                handleSubmit,
-                isSubmitting,
-                setFieldValue,
-                setFieldTouched,
-                status,
-            }) => (
+            {({ errors, handleSubmit, isSubmitting, status }) => (
                 <Form className='account-form__poi-confirm-example' onSubmit={handleSubmit}>
                     <FormBody>
                         <PersonalDetailsForm
-                            errors={errors}
-                            touched={touched}
-                            values={values}
-                            handleChange={handleChange}
-                            handleBlur={handleBlur}
-                            setFieldValue={setFieldValue}
-                            setFieldTouched={setFieldTouched}
                             editable_fields={rest_state.changeable_fields}
-                            is_rendered_for_onfido
+                            is_qualified_for_onfido
+                            side_note={side_note_image}
+                            inline_note_text={
+                                <Localize
+                                    i18n_default_text='To avoid delays, enter your <0>name</0> and <0>date of birth</0> exactly as they appear on your identity document.'
+                                    components={[<strong key={0} />]}
+                                />
+                            }
```

Comment:
Can you create a variable for this like side_note_image so that we can have one line props